Extensions/xkit_patches.js

```diff
@@ -260,6 +260,23 @@ XKit.extensions.xkit_patches = new Object({
 				});
 				return posts;
 			};
+
+			XKit.interface.post_window.state = () => {
+				let post_state = $("#post_state").val();
```

Comment:
I can't find this dom element on the current post_window..... did you confirm it still exists?

Reply:
I didn’t! I’ll take that out.